src/test/java/teammates/test/pageobjects/InstructorFeedbacksPage.java

```diff
@@ -469,6 +469,13 @@ public void verifyUnpublishLinkHidden(String courseId, String sessionName) {
         }
     }
 
+    public boolean verifyCaretDisabled(String courseId, String sessionName) {
+        int sessionRowId = getFeedbackSessionRowId(courseId, sessionName);
+        return !browser.driver.findElement(
+                By.xpath("//tbody/tr[" + (sessionRowId + 1)
+                    + "]//button[contains(@class,'session-caret')]")).isEnabled();
```

Comment:
Since the name contains `verify`, you should do the assertion in this method. Alternatively, rename the method to `is...Disabled`.